### ISSUE TITLE
Confused by box gradients

### ISSUE THREAD

**Issue by Sunjammer** [memononen/nanovg#515]
It's a little unclear to me how the feather/inner/outer arguments of box gradients are intended to work. Feather seems to be an arbitrary value, and it doesn't look to me like a 100x100 box gradient actually interpolates from inner at the center and outer at the very edge. Offsetting the painted rectangle shows the distance field expanding a fair bit beyond the box gradient bounds before i fully arrive at the outer color.

I would really appreciate some help understanding how to reliably draw a box gradient that interpolates completely from inner to outer.

**Comment by memononen** [comment]
The feather works as if the rounded rectangle was box blurred. That is, half the gradient is inside the rectangle and half outside, the center of the gradient is at the rect boundary.

**Comment by Sunjammer** [comment]
So is the feather unit in pixels..?

**Comment by memononen** [comment]
No, but yes :) The units are not pixels, but it is in same units as everything else.
